decred/decred/util/ws.py

```diff
@@ -60,7 +60,7 @@ def on_close(ws):
             if user_close:
                 user_close(ws)
 
-        super().__init__(cleanURL, on_open=on_open, on_close=on_close, *a, **k)
+        super().__init__(cleanURL, on_open=on_open, on_close=on_close, **k)
```

Comment:
Also, positional arguments cannot appear after keyword ones, so this would not have worked anyway.